src/main/java/frc/robot/subsystems/flywheel/FlywheelComponents.java

```diff
@@ -0,0 +1,21 @@
+package frc.robot.subsystems.flywheel;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import frc.robot.hardware.motor.ControllableMotor;
+import frc.robot.hardware.request.IRequest;
+import frc.robot.hardware.signal.InputSignal;
+
+public record FlywheelComponents(
+	String logPath,
+	ControllableMotor topMotor,
+	ControllableMotor bottomMotor,
```

Comment:
cosider changing to left and right (left flywheel, right flywheel)..

Reply:
מכניקה אמרו לי top וbottom כי ברובוט החדש הם באותו צד

Reply:
oh np